Options.py

```diff
@@ -684,6 +684,20 @@ def from_text(cls, text: str) -> Range:
                 return cls.from_any(cls.default)
             else:  # "false"
                 return cls(0)
+        elif (text[0] == "-" and not text[1:].isnumeric()) \
+                or not text.isnumeric():
```

Comment:
This enters the `elif` when it should not: the second half makes the first half completely ignored because "-" is non-numeric. This technically works, I guess?
```suggestion
        elif (text[0] == "-" and not text[1:].isnumeric()) \
                or text[0] != "-" and not text.isnumeric():
```

Reply:
Ah, good catch.

There isn't any style guidance on it, so I tend to prefer parens, but in this case I just used a ternary.